install.sh

```diff
@@ -1,19 +1,59 @@
 #!/usr/bin/env sh
 #
+# Exit codes
+# 0 - Install was successful.
+# 1 - EPERM - Installation failed due to lack of permissions for ~/.local[/share]
+# 2 - mkdir returned an error other than EPERM.
 
 lscolors_data_dir="${XDG_DATA_HOME:-$HOME/.local/share}"
 
+if [[ ! -d $lscolors_data_dir ]]; then
```

Comment:
```suggestion
if [ ! -d $lscolors_data_dir ]; then
```
In POSIX `sh`, `[[` is undefined